cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -1,8 +1,14 @@
 import pytest
 
-from cfme.utils.version import current_version
 from cfme.utils.wait import wait_for
 
+pytestmark = [
+    pytest.mark.uncollectif(lambda appliance: appliance.version < "5.9" and appliance.is_pod,
+                            reason="5.8 pod appliance doesn't support embedded ansible"),
+    pytest.mark.uncollectif(lambda appliance: appliance.version < "5.8"),
```

Comment:
**QUESTION:** Should we have a reason on this one?

Reply:
we definitely should.  I suspect ansible support  was added in 5.8.  But I'm not 100% sure.  I'd ask test owners.

Reply:
@lcouzens, could you confirm whether I'm right ? btw, would you like to add reason everywhere in ansible tests?

Reply:
Yes embedded ansible was first added in 5.8, as for reasons I don't mind.